app/pages/index.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <div>
     <section class="grid-cols-2 mb-16 md:grid home-top">
-      <div class="bg-white">
+      <div v-if="newsListState" class="bg-white">
         <NewsHome :news="sortNews[newsStart]" @changenews="changeIt($event)" />
       </div>
-      <div>
+      <div v-if="sortShowsForSchedule">
```

Comment:
For me it would be simpler to directly see what the conditional is doing instead of putting the method name here that we use below as well, does this go against some best practices? What do you think?

Can we just write this conditional rendering directly `v-if="this.arcsishows"` ? Same for `newsList` & `newsListState`

Reply:
it is needed to check if this computed object exists at all, before trying to render. otherwise it throws an error if it is still not calculated before rendering
it is a general practice in vue/nuxt, yes